parachain/runtime/tests/mock/mod.rs

```diff
@@ -384,10 +408,40 @@ impl CoreVaultData {
     }
 }
 
+#[derive(Debug, PartialEq, Clone)]
+pub struct CoreOperatorData {
+    pub nominators: BTreeMap<AccountId, Nominator<AccountId, BlockNumber, u128>>,
```

Comment:
Your tests with the ParachainState are quite long, and because of that, not really the most readable. We need to strike a balance between the clarity and the completeness of the tests. So far, we limited the ParachainState to accounting information. Note that e.g. CoreVaultData is not just storing the Vault type as specified in vault_registry - it abstracts over it, such we don't include irrelevant data (e.g. `Vault.wallet`), and that we can add information that is stored in other crates (e.g. locked griefing collateral). 

So, what I would suggest is to make this `pub nominators: BTreeMap<AccountId, CoreNominatorData>`, where `CoreNominatorData` contains only accounting information (at first glance, I think that would be collateral and collateral_to_be_withdrawn). What do you think?